deploy/operator/ztp/deploy_spoke_cluster.sh

```diff
@@ -89,5 +89,8 @@ echo "Cluster installation has been stopped (either for good or bad reasons)"
 wait_for_condition "agentclusterinstall/${ASSISTED_AGENT_CLUSTER_INSTALL_NAME}" "Completed" "1m" "${SPOKE_NAMESPACE}"
 echo "Cluster has been installed successfully!"
 
+echo "Test webhook - try to update ACI after install"
+oc -n ${SPOKE_NAMESPACE} patch agentclusterinstall.extensions.hive.openshift.io ${ASSISTED_AGENT_CLUSTER_INSTALL_NAME} -p '{"spec":{"provisionRequirements":{"controlPlaneAgents":"7"}}}' --type merge
+
```

Comment:
It isn't really clear why this is being changed as well. Please add to the description the reason the base CI script is being changed here. This will effect all CI jobs so we need to understand that is actually desirable.

Reply:
This is still WIP.
As we discussed some time ago, this is the only place where I can test the webhook.
The idea is to attempt to change the ACI after it is installed and verify that it will be revoked.

I need to check the return value and validate that it fails.

Reply:
You will need to make a new CI job for this test case. I suggest removing this until you understand how to add this check without breaking all the existing jobs with this change.